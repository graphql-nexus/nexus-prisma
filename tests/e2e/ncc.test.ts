import * as Path from 'path'
import { setupTestProject } from '../__helpers__/testProject'

it('works with ncc', async () => {
  const testProject = setupTestProject({
    fixture: Path.join(__dirname, 'fixtures/ncc'),
  })

  await testProject.runOrThrowNpmScript(`build`)

  const result = await testProject.runOrThrowNpmScript(`start:dist`)

  expect(result.stdout).toMatchSnapshot()
})
