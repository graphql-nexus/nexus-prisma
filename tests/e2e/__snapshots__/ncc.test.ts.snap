// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works with ncc 1`] = `
"type Foo {
  id: ID!
}

type Query {
  foos: [Foo]
}"
`;
